### ISSUE TITLE
Add LSTM to pipeline

### ISSUE THREAD

**Issue by malawada** [louisccc/av_av#17]
add lstm for learning label across multiple scenes

**Comment by louisccc** [comment]
This is done by having torch.nn.lstm.
